.github/workflows/unit_tests.yml

```diff
@@ -66,6 +66,25 @@ jobs:
           command: test
           args: --lib
 
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.8'
+
+      - name: Install pylint and pycodestyle
+        run: |
+          python -m pip install --upgrade pip
+          pip install pylint pycodestyle
+
+      - name: Run pylint and pycodestyle
+        run: |
+          find . -type f -name "*.py" | xargs pylint \
+          --disable=missing-docstring,too-many-instance-attributes,duplicate-code,import-error \
+          --good-names-rgxs=^[_a-z][_a-z0-9]?$
+
+          find . -type f -name "*.py" | xargs pycodestyle \
```

Comment:
We can also make a config for pycodestyle.

Reply:
-> .pycodestyle file